Firmware/RTK_Everywhere/MQTT_Client.ino

```diff
@@ -480,7 +480,9 @@ void mqttClientStop(bool shutdown)
     if (shutdown)
     {
         mqttClientSetState(MQTT_CLIENT_OFF);
-        settings.enablePointPerfectCorrections = false;
+        //settings.enablePointPerfectCorrections = false;
+        //^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Why? This means PointPerfect Corrections
+        //cannot be restarted without opening the menu or web configuration page...
```

Comment:
Agreed. I've seen this before and overwrites a user setting when it should not. I've had to remove the settings mod code and add an override. See sdCardForcedOffline in RTK Firmware.